src/core/utils.js

```diff
@@ -737,3 +737,35 @@ export function getLinkTargets(elem) {
     return result;
   }, []);
 }
+
+/**
+ * Changes name of a DOM Element
+ * @param {Element} elem element to rename
+ * @param {String} newName new element name
+ * @returns {Element} new renamed element
+ */
+export function renameElement(elem, newName) {
+  const newElement = elem.ownerDocument.createElement(newName);
+
+  // copy attributes
+  for (let i = 0, n = elem.attributes.length; i < n; i++) {
+    const attr = elem.attributes[i];
+    try {
+      newElement.setAttributeNS(attr.namespaceURI, attr.name, attr.value);
```

Comment:
Nit: I know this is copy/paste... but we can just jus `setAttribute`. We don't support XHTML anymore.

Reply:
Removing this exception handling as we have `renameElement` as an internal function, and we rename only specific elements. So, we never get this exception :crossed_fingers:.